md/software-extensibility.md

```diff
@@ -99,6 +99,11 @@ To [implement an event handler](event-handlers.md), you need to provide a Java c
 
 ### BonitaStudioBuilder
 
+::: warning
+**Important note**: as of Bonita 7.7, the BonitaStudioBuilder tooling (headless studio build) has been deprecated. See the
+[Release Notes page](release-notes.md) for more information
+:::
```

Comment:
```suggestion
:::We strongly encourage you to use the LA builder included in the tooling suite of Bonita Continuous Delivery add-on. One added-value is that LA builder does not need a studio to be installed.
```

Reply:
applied directly inside the warning block